azure-pipelines.yml

```diff
@@ -110,11 +110,7 @@ jobs:
     displayName: 'E2E Test - CDN (only runs on release builds)'
     # This test only runs after deployment from a release branch and the new CDN version has been deployed
     # This check will run on the PR to merge the release branch back into main
-    condition: and(
```

Comment:
This is changed to trigger release process and test if E2E test also work successfully on releasing process. It will be reverted back once everything else is good to be checked in.

Reply:
Reverted.